keras/layers/preprocessing/text_vectorization.py

```diff
@@ -174,6 +174,8 @@ class TextVectorization(base_preprocessing_layer.PreprocessingLayer):
       sparse: Boolean. Only applicable to `"multi_hot"`, `"count"`, and
         `"tf_idf"` output modes. If True, returns a `SparseTensor` instead of a
         dense `Tensor`. Defaults to False.
+      encoding: Optional. A string representing the encoding to use. Defaults to
```

Comment:
Likewise

Reply:
updated the encoding detail in docstring